tests/unit/sagemaker/model/test_neo.py

```diff
@@ -362,59 +362,35 @@ def test_compile_with_tensorflow_neo_in_ml_inf(session):
     )
 
 
-def test_compile_validates_framework_version(sagemaker_session):
-    sagemaker_session.wait_for_compilation_job = Mock(
-        return_value={
-            "CompilationJobStatus": "Completed",
-            "ModelArtifacts": {"S3ModelArtifacts": "s3://output-path/model.tar.gz"},
-            "InferenceImage": None,
-        }
-    )
+@pytest.mark.parametrize(
+    "target,framework,fx_version,expected_fx_version",
+    [
+        ("ml_c4", "pytorch", "1.6", "1.6"),
+        ("rasp3b", "pytorch", "1.6.1", "1.6"),
+        ("amba_cv2", "pytorch", "1.6.1", None),
+        ("ml_c4", "tensorflow", "1.15.1", "1.15"),
+        ("ml_c4", "tensorflow", "2.15.1", "2.15"),
```

Comment:
For completeness add gpu targets as well

Reply:
will do